source/stylesheets/_navigation.scss

```diff
@@ -0,0 +1,210 @@
+
+// ==== main nav =====
+.nav-title{
+	color: $text-color-lighter;
+	font-weight: $medium-weight;
+	border-bottom: none;
+	padding: $medium-padding;
+}
+
+ul{
+	list-style-type: none;
+    @extend .center-object;
+}
+
+li {
+	float: left;
+}
+
+li a {
+    display: block;
+    text-align: center;
+    padding-left: 2.3em;
+    padding-right: 2.3em;
+    text-decoration: none;
+    border-bottom: none;
+    font-size: $subhead-size;
+}
+
+// selected navigation bar element 
+.active {
+ 	color: $emphasis_color-medium;
+ 	font-weight: $medium-weight;
+
+}
+
+.header-shadow{
+	box-shadow: 0 .1em .2em #E8E8E8;
+	width: 100%;
+	height: 5em;
+	property: float;
+	z-index: 200;
+	background-color: white;
+	position: relative;
+}
+
+.white-text{
+	color:white;
+}
+
+// moves orange button into upper nav bar on mobile
+.position-button{
+	position: absolute;
+	top: 1em;
+	right: $medium-padding;
+	}
+
+
+// Aligns nav right when in desktop. Centers  nav in mobile
+.align-nav-right{
+	 display:flex;
+	 align-items: center;
+	 justify-content: center;
+}
+
+// Aligns title left when in desktop or very small screen sizes. Centers in mid range sizes
+.align-title-left{
```

Comment:
Similar comment as in other areas.. but include the query styles here. Also.. generally we'll lay things out where the base style should be the mobile one, and break-points override those as the screen gets bigger. Can discuss more in person.